docs/content/en/docs/concepts/tracing-policy.md

```diff
@@ -1408,6 +1408,21 @@ There are different types supported for each operator. In case of `matchArgs`:
 * Prefix
 * Postfix
 * Mask
+* GreaterThan (aka GT)
+* LessThan (aka LT)
+* SPort
+* NotSPort
+* SPortPriv
+* NotSPortPriv
+* DPort
+* NotDPort
+* DPortPriv
+* NotDPortPriv
+* SAddr
+* NotSAddr
+* DAddr
+* NotDAddr
+* Protocol
```

Comment:
Do you think you could document that?

Since this is adding a lot of specialized operators only for one arg type, it would be nice if people can understand the shortcuts like `SPortPriv` without having to read the commit message.

It looks like we could document the operators more since they have different meanings depending on the context (i.e. equal on struct file means path equality, this kind of stuff), especially those that only apply to very specific types.

Reply:
Yep, I can do that now.